src/test/java/ca/bc/gov/ag/efax/mail/service/parser/SucceededVisitorTest.java

```diff
@@ -0,0 +1,42 @@
+package ca.bc.gov.ag.efax.mail.service.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.Test;
+
+import ca.bc.gov.ag.efax.BaseTestSuite;
+import ca.bc.gov.ag.efax.mail.model.DocumentDistributionMainProcessProcessResponseDecorator;
+
+public class SucceededVisitorTest extends BaseTestSuite {
+
+    @Test
+    void testApply_Success() throws Exception {
+        SucceededVisitor visitor = new SucceededVisitor();
+        String subject = "Message Succeeded: 2505551234 (2505551234@somewhere.com) on 8/13/2021 at 12:38:15 PM";
+        String body = "Your fax which was sent to 2505551234@somewhere.com at 2505551234 with subject <jobId>1234</jobId><uuid>8e3b47c4-60b6-43e7-81ee- c9ac5930e49d</uuid> was delivered successfully";
```

Comment:
Not that it is important but a valid guid cant have a space.

Reply:
ah yes. fixed. thanks!